benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -847,6 +940,12 @@ void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainP
         frame.cmd->BindGraphicsPipeline(mQuadsPipelines[pFullscreenQuadsType->GetIndex()]);
         frame.cmd->BindVertexBuffers(1, &mFullscreenQuads.vertexBuffer, &mFullscreenQuads.vertexBinding.GetStride());
 
+        if (pFullscreenQuadsType->GetIndex() == static_cast<size_t>(FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_TEXTURE)) {
+            grfx::DescriptorSet* sets[1] = {nullptr};
```

Comment:
i guess we could put `mFullscreenQuads.descriptorSets.at(GetInFlightFrameIndex()` directly in the initialize list? or maybe no need to create this array, just pass `&mFullscreenQuads.descriptorSets.at(GetInFlightFrameIndex())` directly?

Reply:
Sure, changed this and other cases